CMakeLists.txt

```diff
@@ -1,15 +1,10 @@
 ######################################################################
 # CMake version and policies
 ######################################################################
-cmake_minimum_required(VERSION 3.15.0)
-
-# CMP0074: CMake find_package will use <PackageName>_ROOT CMake variable
-# and environment variable in search path.
-# various Find<Package>.cmake modules may not follow this policy
-# CMP0075: check_include_file_cxx and other follow CMAKE_REQUIRED_LIBRARIES
-# when doing link check
-cmake_policy(SET CMP0074 NEW)
```

Comment:
I don't believe this can be dropped.  My reading of this is as of 3.23 if the policy is not set you get a warning and OLD not NEW.
Also seems true of CMP0075

Reply:
See the added comment right after ` cmake_minimum_required(VERSION 3.17.0) `
CMP0075 is known to your 3.23 and introduced before 3.17. Thus NEW is set.